README.md

```diff
@@ -783,27 +783,27 @@ To reduce storage costs for these artifacts, you can set the `artifact-retention
 
 
 
-# Gradle Enterprise plugin injection
+# Develocity plugin injection
 
-The `gradle-build-action` provides support for injecting and configuring the Gradle Enterprise Gradle plugin into any Gradle build, without any modification to the project sources.
+The `gradle-build-action` provides support for injecting and configuring the Develocity Gradle plugin into any Gradle build, without any modification to the project sources.
 This is achieved via an init-script installed into Gradle User Home, which is enabled and parameterized via environment variables.
 
 The same auto-injection behavior is available for the Common Custom User Data Gradle plugin, which enriches any build scans published with additional useful information.
 
-## Enabling Gradle Enterprise injection
+## Enabling Develocity injection
 
-In order to enable Gradle Enterprise for your build, you must provide the required configuration via environment variables. 
+In order to enable Develocity for your build, you must provide the required configuration via environment variables. 
 
 Here's a minimal example:
 
 ```yaml
-name: Run build with Gradle Enterprise injection
+name: Run build with Develocity injection
   
 env:
-  GRADLE_ENTERPRISE_INJECTION_ENABLED: true
-  GRADLE_ENTERPRISE_URL: https://ge.gradle.org
-  GRADLE_ENTERPRISE_PLUGIN_VERSION: 3.16.1
-  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_KEY }} # Required to publish scans to ge.gradle.org
+  DEVELOCITY_INJECTION_ENABLED: true
+  DEVELOCITY_URL: https://develocity.gradle.org
```

Comment:
Are we planning to rename this server soon? As far as I can tell it's still `ge.gradle.org`.
Since this is just an example, perhaps we should instead use something like `https://develocity.your-server.com` rather than a real-world server.

Reply:
I think having a dummy server like `https://develocity.your-server.com` is much preferred here as this is just an example.
I've updated this accrodingly.